SPECS-EXTENDED/azl-otel-collector/azl-otel-collector.spec

```diff
@@ -0,0 +1,63 @@
+%global debug_package   %{nil}
+
+Summary:        Azure Linux OpenTelemetry Collector Distribution
+Name:           azl-otel-collector
+Version:        0.123.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://github.com/microsoft/azl-otel-collector
+Source0:        https://azurelinuxsrcstorage.blob.core.windows.net/sources/core/azl-otel-collector-0.123.0-3.tar.gz
+Source1:        %{name}-%{version}-vendor-1.tar.gz
+Source2:        azl-otel-collector.service
+BuildRequires:  golang
+BuildRequires:  systemd-rpm-macros
+# Include the smartmontools package needed by the smartdata receiver in the collector
+Requires:       smartmontools
+Requires(post): systemd
+Conflicts:      azl-otel-collector
```

Comment:
what does `Conflicts` mean here? Isn't this the same name as that of this package?

Reply:
removed, initially added it to avoid multiple instances of the azl-otel-collector package but realized we don't need to enforce that